src/render/tutorials/caching-http-requests/2_0.html.md

```diff
@@ -0,0 +1,322 @@
+    ports:
+      - "3000:8080"
+    networks:
+      - knotnet
+
+  knotx:
+    image: knotx-example/api-cache:latest
+    command: ["knotx", "run-knotx"]
+    ports:
+      - "8092:8092"
+      - "18092:18092"
+    networks:
+      - knotnet
+```
+
+First and foremost it specifies Docker virtual network in which our services will be deployed. Then we have some service configs.
+The first one is `webapi`
+- `image` - docker image is provided here, 
+- `volumes` - directories to which Docker will have access, in our case it's directory where all Wiremock configs are stored,
+- `ports` - port accessible from `localhost` is specified on the left side, and a port available in the virtual environment is specified on the right 
+- `networks` - specifies in which docker network the service will be deployed.
+
+Then we have `knotx` service specification. It's very similar to the previous one, so only one property is worth explaining. 
+- `command` - this property is used to specify Knot.x starting commands.
+
+Next step in our configuration is to set `rootProject.name` in `settings.gradle.kts` (also at the project level directory),
+also make sure that `healh-check` is included 
+```gradle
+rootProject.name = "api-cache"
+
+include("health-check")
+
+project(":health-check").projectDir = file("modules/health-check")
+``` 
+
+Let's define the custom Docker image name for Knot.x instance. We can specify it by the `docker.image.name` property in the `./gradle.properties` file:
+```gradle
+version=2.0.0-SNAPSHOT
+knotx.version=2.0.0
+knotx.conf=knotx
+docker.image.name=knotx-example/api-cache
+```
+
+## Wiremock 
+Wiremock is simply used for mocking server, we don't have to create real REST API, for our training purposes it's enough.
+To do it let's create following directories on the project level directory:
+- `common-services/webapi/__files/`
+- `common-services/webapi/mappings/`
+
+Into first directory please paste `product.json` file with following body:
+```json
+{
+  "id": 21762532,
+  "url": "http://knotx.io",
+  "label": "Product"
+}
+```
+
+And into second directory please paste `product.json` file with following body:
+```json
+{
+  "request": {
+    "method": "GET",
+    "url": "/product/id"
+  },
+  "response": {
+    "status": 200,
+    "fixedDelayMilliseconds": 100,
+    "bodyFileName": "product.json"
+  }
+}
+```
+
+Make sure that in `api-cache.yml` property `volume` properly points to `webapi` directory. And that's all about Wiremock.
+
+## Build and run
+From project level directory execute following commands which will build project and run Docker Swarm:
+```shell script
+$ gradlew clean build
+$ docker swarm init
+$ docker stack deploy -c api-cache.yml api-cache
+```
+After successful execution you should be able to see http response at [localhost:8092/product/id](localhost:8092/product/id)
+
+## Summary
+In this tutorial we've configured in-memory cache and we've shown how easily we can replace it with another one.
```

Comment:
What do you think about this:
> In this tutorial, we used Knot.x as a simple API gateway providing very flexible API caching functionality. With [Configurable Integrations](https://knotx.io/blog/configurable-integrations/) we
achieved it without writing any line of code, using only Knot.x built-in features. With [Actions](https://github.com/Knotx/knotx-fragments/tree/2.0.0/handler#actions) and [Behaviours](https://github.com/Knotx/knotx-fragments/tree/2.0.0/handler#behaviours), we've focused on creating it in a declarative way instead of implementing caching functionality on our own. Now you'll understand how quickly you can "decorate" your existing API with cross-cutting functionalities such as caching or circuit breaker.

Reply:
looks better, applied